src/components/AppCatalog/CatalogList/CatalogList.js

```diff
@@ -1,9 +1,19 @@
+import styled from '@emotion/styled';
 import DocumentTitle from 'components/shared/DocumentTitle';
 import PropTypes from 'prop-types';
 import React from 'react';
+import Well from 'UI/Well';
 
 import CatalogRepo from './CatalogRepo';
 
+const Wrapper = styled.div`
```

Comment:
Is the new `div` required here? if its only to get a way to wrap the `h1` styles, I think it would be cleaner to define the `h1` styles in a styled component